[docs] Fix typo in "Up and Running" (#5031)
expo/expo: docs/pages/versions/unversioned/workflow/up-and-running.md

#### Manually reloading the app

> - If you've followed the above steps and live reload **still** doesn't work, [post to Expo Forums](https://forums.expo.io/c/help) to send us a support request. Until we resolve your reported issue, you can either shake the device and press `Reload`, or use one of the following tools which work both with and without development mode.

Reviewer: ```suggestion
- If you've followed the above steps and live reload **still** doesn't work, [post to Expo Forums](https://forums.expo.io/c/help) to send us a support request. Until we resolve the issue for you, you can either shake the device and press `Reload`, or use one of the following tools which work both with and without development mode.
```
